### DISCUSSION TITLE
What do I need for a 32-bit chroot?

### DISCUSSION THREAD

**Discussion by hack-wizard** [ptitSeb/box86#602] [Q&A]
So... I find your readme somewhat vague on a few points:
- I know what a chroot is but you may want to consider that a large number of your prospective users likely do not.  It's not exactly something a newcomer who just wants to play x86 games on their 64-bit arm install would necessarily know.  A brief explanation with basic steps would be extremely helpful here.
- Just saying "you need a chroot with 32 bit libraries" is like saying "make me a sandwich with stuff on it."  Could you please elaborate as to **which** libraries are needed?
- is there an ideal way this should be constructed?  Can I just use a subdirectory with an extracted OS image?  An image file mounted to a loopback device?  Does it need to actually be on a separate partition/drive?
- do we need to mount special filesystems (like proc or sysfs) for this to work right?
- Are there other useful tools or software to include?  For instance does it need a working window management instance?
- Please explain in detail how one would "fool" build scripts and thinga like appimages to install using Box86

Sorry if this seems rather critical, and maybe these feel like extremely dumb questions, but I can't help but feel hopelessly lost after reading the docs, so I'd appreciate a little clarification on the above.

**Comment by WheezyE** [comment]
Thank you for your suggestions. box86 is pretty small in file size and easy
to download and you can experiment with it pretty easily if you like. If
you find out anything to document, please do and make a pull request. Seb
has put in a ton of work on this emulator (which runs faster than qemu,
FEX, and ExaGear) and given it to the community for free. If you don’t want
to use box86 either, you don’t have to.

**Reply by hack-wizard** [reply]
I'm going to be trying a few things this weekend, if it works out I'll probably post/suggest them.

**Reply by Wooty-B** [reply]
I have a WIP guide I am writing for my dev board, but this page should apply for a 64bit system:

https://github.com/Wooty-B/LX2K_Guide/blob/main/Setup-Armhf-Chroot.md

If you follow word-for-word, change the references to the LX2160A under Box64 section. This is for a Debian based system.
